test.sh

```diff
@@ -89,10 +89,11 @@ function test_project {
   project_dir=$1
   version=$2
   header_text "performing tests in dir $project_dir for project version v$version"
-  cd test/$project_dir
-  tar -zxf ../vendor.v$version.tgz
+  vendor_tarball=$(pwd)/testdata/vendor.v$version.tgz
+  cd testdata/$project_dir
+  [[ -e ${vendor_tarball} ]] && tar -zxf $vendor_tarball
```

Comment:
we should add a comment saying `v2` uses `go modules`, so vendor tarball will not be there for `v2`.